src/createServer.js

```diff
@@ -1,11 +1,233 @@
+/* eslint-disable function-paren-newline */
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const router = express.Router();
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+  const mockUsers = [];
+  const mockExpenses = [];
+
+  app.use(cors());
+  app.use(express.urlencoded({ extended: true }));
+  app.use(express.json());
+  app.use('/', router);
+
+  app.get('/users', (req, res) => {
+    if (mockUsers.length === 0) {
+      return res.send([]);
```

Comment:
Is this _if_ statement needed?
Removing it will also send(mockUsers) which is an empty array by default